components/home-components/what-we-do.tsx

```diff
@@ -1,7 +1,34 @@
 import { HomePageType } from '@/lib/interface'
-import React from 'react'
 import { WrapperLarge } from '../wrapper'
 
 export default function WhatWeDo({ data }: HomePageType) {
-  return <WrapperLarge>What We Do </WrapperLarge>
+  return (
+    <WrapperLarge className="bg-brown-bg place-items-center gap-1.5 text-center">
+      <div className="text-2xl text-red py-4">
+        {data.homepage.whatWeDoHeading}
+      </div>
+
+      <h2 className="text-7xl font-bold">{data.homepage.whatWeDoTitle}</h2>
+
+      <div className="grid mt-10 xl:grid-cols-4 gap-5 md:grid-cols-2 gap-5">
+        {data.homepage.whatWeDoCards.map((x, index) => {
```

Comment:
why "x" and not "whatWeDoCard" or "card"?

Reply:
I fixed that.